src/main/java/driver/LocalDriverStategy.java

```diff
@@ -0,0 +1,79 @@
+package driver;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.jetbrains.annotations.Contract;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import utility.PropertyProvider;
+
+import java.util.concurrent.TimeUnit;
+
+
+public class LocalDriverStategy {
```

Comment:
If you are adding "Strategy" to class name, than you say that this class is a part of strategy pattern. In this case I don't see strategy pattern. Your class is close to Singleton pattern. Rename your class